tests/installation/ipxe_install.pm

```diff
@@ -131,26 +131,43 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+
 sub run {
     my $self = shift;
 
     poweroff_host;
 
-    #virtualization tests use a static ipxe configuration file
-    set_bootscript unless get_var('VIRT_AUTOTEST');
+    #virtualization tests use a static ipxe configuration file in O3
+    set_bootscript unless get_var('VIRT_AUTOTEST') && is_tumbleweed;
 
     set_pxe_boot;
 
     poweron_host;
 
+    select_console 'sol', await_console => 0;
+
+    #use ipxe bootloader in O3
+    #it is static menu and choose the TW entry to start installation
+    if (get_var('VIRT_AUTOTEST') && is_tumbleweed) {
+        ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
+        assert_screen('o3-ipxe-menu', 300);
+        send_key 't';
+        assert_screen([qw(load-linux-kernel load-initrd)], 240);
```

Comment:
Asserting 'load-linux-kernel' and  'load-initrd' at the same time does not make too much sense, because these two can not appear on the screen at the same time. They appear in order, firstly 'kernel' then 'initrd'. So assert_screen will always  return on detection of 'kernel'. It nearly has no chance of finding 'initrd'. Suggest
```
assert_screen('load-initrd', 240);
```
or 
```
assert_screen('load-kernel', 240);
assert_screen('load-initrd', 240);
```
or
skip detecting these two needles, but you can record the screen by using check_screen ?

Reply:
In actural, becuase of ipmi console problem or something else, the 'load-kernel' or 'load-initrd' are not always occur at the same time, f.g. http://10.67.129.102/tests/306#step/ipxe_install/4, in this case only 'load-initrd' is present. there was some cases only 'load-kernel' occured. So the test continues with either of them.

Reply:
I think
```
assert_screen('load-kernel-and-initrd', 240);
```
is better.
But if you think this will cause instability, then I am fine with current code.